pages/linux/apt.md

```diff
@@ -1,38 +1,38 @@
 # apt
 
-> Package management utility for Debian based distributions.
-> Recommended replacement for `apt-get` when used interactively in Ubuntu versions 16.04 and later.
+> Command-line package manager for Debian-based distributions.
+> Intended as a user-friendly alternative to `apt-get` for interactive use.
```

Comment:
How is `apt` more user-friendly than `apt-get`? From what I remember from the time I used it, they were pretty much the exact same.